rts/Game/UI/SelectionKeyHandler.cpp

```diff
@@ -432,6 +432,27 @@ void CSelectionKeyHandler::DoSelection(std::string selectString)
 			return;
 		} break;
 
+		case hashString("SelectOneNearestMouse"): {
+			if(selection.empty())
+				return;
+
+			const float gndDist = CGround::LineGroundCol(camera->GetPos(), camera->GetPos() + mouse->dir * camera->GetFarPlaneDist(), false);
+			float3 mousePosition = camera->GetPos() + mouse->dir * gndDist;
+
+			CUnit* closest = nullptr;
+			float closestDistance = 0;
+			for(auto ui = selection.begin(); ui != selection.end(); ui++) {
```

Comment:
Space after `for`
Also prefer something like `for (const auto* su : selection)` for trivial loops.

Reply:
done, however I cannot make it const as `selectedUnitsHandler.AddUnit` takes a non-const pointer :-/

Reply:
Yeah non-const is fine too